content/en/blog/falco-0-36-0/index.md

```diff
@@ -0,0 +1,139 @@
+
+### Falco WASM
+
+Flaco is excited to introduce its latest addition: the WebAssembly target. This new target has been developed exclusively for the Falco Playground using [Emscripten](https://emscripten.org/docs/tools_reference/emcc.html), where it brings essential core functionalities to the forefront. These functionalities include a rule compiler and the ability to reproduce events from capture files.
+It’s worth noting that certain features, such as kernel modules and Kubernetes support, have been intentionally omitted from this wasm target. This omission is due to the inherent limitations of running these features within a web browser environment. falco.wasm can be found as a [github artifact](https://github.com/falcosecurity/falco/actions/workflows/ci.yml) in the latest workflow.
+
+### Falco Playground
+
+Falco playground is simple web application where you can create, edit and validate [falco rules](https://github.com/falcosecurity/rules). This is a quick solution for users wanting to easily check the accuracy of their custom rules. This application is completely client side and doesn’t make calls to any backend server. It leverages the power of [WebAssembly](https://webassembly.org/) to test your rules.
+
+### Falco Kernel Testing Framework
+
+Falco supports a large number of Linux kernels. And the truth is, in order to test this kind of functionality you have to start an equally large number of live Linux systems and load the driver there. This is absolutely not easy to do and just taking a look at the [task list](https://github.com/falcosecurity/libs/issues/1191) for such an endeavor gives you an idea of the complexity required. The results are awesome: you can find a matrix of kernels that are continuously tested for [x86_64](https://falcosecurity.github.io/libs/matrix_X64/) and [ARM](https://falcosecurity.github.io/libs/matrix_ARM64/) as well!
+
+## Try it out
+
+It’s time to try out the new release! Here are some pointers for getting started with Falco:
+
+* [Container Images](/docs/getting-started/running/#docker)
+  * `falco` ([DockerHub](https://hub.docker.com/r/falcosecurity/falco), [AWS ECR Gallery](https://gallery.ecr.aws/falcosecurity/falco))
+  * `falco-no-driver` ([DockerHub](https://hub.docker.com/r/falcosecurity/falco-no-driver), [AWS ECR Gallery](https://gallery.ecr.aws/falcosecurity/falco-no-driver))
+  * `falco-driver-loader` ([DockerHub](https://hub.docker.com/r/falcosecurity/falco-driver-loader), [AWS ECR Gallery](https://gallery.ecr.aws/falcosecurity/falco-driver-loader))
+* [CentOS/Amazon Linux](/docs/getting-started/installation/#centos-rhel)
+* [Debian/Ubuntu](/docs/getting-started/installation/#debian)
+* [openSUSE](/docs/getting-started/installation/#suse)
+* [Linux binary package](/docs/getting-started/installation/#linux-binary)
+
+## What next? 🔮
+
+The community is active on many topics and we hope to deliver great features and many stability fixes once again during the next release cycle!
+
+<!--
```

Comment:
here we could have the k8s meta collector as a plugin

Reply:
also 
* `falco-driver-loader` implemented into `falcoctl`
* syscall 32 bit detection
* other improvements in the rules framework and UX

Reply:
Thanks!